Source/Core/Core/Boot/Boot.cpp

```diff
@@ -351,28 +351,9 @@ bool CBoot::BootUp()
       DVDInterface::SetVolumeName(_StartupPara.m_strDefaultISO);
     }
 
-    // Set up MSR and the BAT SPR registers.
-    UReg_MSR& m_MSR = ((UReg_MSR&)PowerPC::ppcState.msr);
-    m_MSR.FP = 1;
-    m_MSR.DR = 1;
-    m_MSR.IR = 1;
-    m_MSR.EE = 1;
-    PowerPC::ppcState.spr[SPR_IBAT0U] = 0x80001fff;
-    PowerPC::ppcState.spr[SPR_IBAT0L] = 0x00000002;
-    PowerPC::ppcState.spr[SPR_IBAT4U] = 0x90001fff;
-    PowerPC::ppcState.spr[SPR_IBAT4L] = 0x10000002;
-    PowerPC::ppcState.spr[SPR_DBAT0U] = 0x80001fff;
-    PowerPC::ppcState.spr[SPR_DBAT0L] = 0x00000002;
-    PowerPC::ppcState.spr[SPR_DBAT1U] = 0xc0001fff;
-    PowerPC::ppcState.spr[SPR_DBAT1L] = 0x0000002a;
-    PowerPC::ppcState.spr[SPR_DBAT4U] = 0x90001fff;
-    PowerPC::ppcState.spr[SPR_DBAT4L] = 0x10000002;
-    PowerPC::ppcState.spr[SPR_DBAT5U] = 0xd0001fff;
-    PowerPC::ppcState.spr[SPR_DBAT5L] = 0x1000002a;
     if (dolWii)
       HID4.SBE = 1;
-    PowerPC::DBATUpdated();
-    PowerPC::IBATUpdated();
+    SetupBAT(/*is_wii*/ true);
```

Comment:
Hm, should we call this with `dolWii` (or perhaps `_StartupPara.bWii`) instead?
It probably doesn't make a difference in practise because the GC doesn't have `SPR_{I,D}BAT{4,5}{U,L}` and thus shouldn't look at them, but for accuracy, don't initialize what we don't need?
I dunno tbh, just something to look at.